src/services/cost-explorer/budget/budget-detail/modules/budget-info/BudgetDetailInfo.vue

```diff
@@ -86,6 +86,14 @@ export default {
         },
     },
     setup() {
+        // * INIT * //
```

Comment:
```suggestion
        /* Init */
```

자바스크립트 주석 종류로

```
// single line comment

/* 
multi line comment 
*/ 
```
이렇게 두 가지가 있고, 위에 작성하신 것은 두 가지가 혼합된 것 같네요.